app/views/kaminari/_first_page.html.erb

```diff
@@ -0,0 +1,11 @@
+<%# Link to the "First" page
+  - available local variables
+    url:           url to the first page
+    current_page:  a page object for the currently displayed page
+    total_pages:   total number of pages
+    per_page:      number of items to fetch per page
+    remote:        data-remote
+-%>
+<span class="govuk-pagination__prev">
```

Comment:
I think using `<span>` elements for each of the items here is slightly problematic. If possible these should be `<li>` elements. 

Also at smaller sizes the `govuk-pagination__prev` and `govuk-pagination__next` classes add some slightly awkward padding when used for the first and last items as well.

Reply:
I've copied the implementation from Whitehall, so this now works identically to that and uses `li` elements.